src/httpcommon.cpp

```diff
@@ -1,5 +1,9 @@
 #define BOOST_BIND_GLOBAL_PLACEHOLDERS
 
+#ifdef _WIN32
+#define CURL_STATICLIB
+#endif
```

Comment:
I'm confused why this is needed since your changes don't affect that. I guess it's an isolated build issue? But if it improves the build I'm okay with keeping it.

Reply:
This define is probably necessary, but as of the merging of #216, it's already added via cmake through `${CURL_STATIC_CFLAGS}`: https://github.com/LizardByte/Sunshine/blob/nightly/CMakeLists.txt#L94

This should therefore be unnecessary on latest nightly.